docs/auth/index.md

```diff
@@ -1,99 +1,121 @@
 ---
 id: index
-title: User Authentication and Authorization in Backstage
-description: Documentation on User Authentication and Authorization in Backstage
+title: Adding Authentication
+description: How to add authentication to a Backstage application
 ---
 
-## Summary
```

Comment:
I moved this original content to a `using-auth` file, diff not super helpful here.